cfme/containers/replicator.py

```diff
@@ -42,6 +45,19 @@ def get_detail(self, *ident):
         self.load_details(refresh=True)
         return details_page.infoblock.text(*ident)
 
+    @classmethod
+    def get_random_instances(cls, provider, count=1, appliance=None):
```

Comment:
**OPTIONAL** i see a lot of duplication of this randomization pattern, please take a look at moving the randomization/number limitation logic to a helper function and a look at `itertools.islice` for that helper